code/modules/projectiles/guns/manufacturer/frontier_import/ballistics.dm

```diff
@@ -0,0 +1,229 @@
+		ATTACHMENT_SLOT_STOCK = 1
+	)
+	slot_offsets = list(
+		ATTACHMENT_SLOT_MUZZLE = list(
+			"x" = 32,
+			"y" = 23,
+		),
+		ATTACHMENT_SLOT_STOCK = list(
+			"x" = -5,
+			"y" = 18,
+		)
+	)
+
+	default_attachments = list(/obj/item/attachment/foldable_stock/spitter)
+
+	gun_firemodes = list(FIREMODE_FULLAUTO)
+	default_firemode = FIREMODE_FULLAUTO
+
+
+
+/obj/item/ammo_box/magazine/spitter_9mm
+	name = "spitter pistol magazine (9mm)"
+	desc = "A thin, 30-round magazine for the spitter machine pistol. These rounds do okay damage, but struggle against armor."
+	icon_state = "spitter_mag-1"
+	base_icon_state = "spitter_mag"
+	ammo_type = /obj/item/ammo_casing/c9mm
+	caliber = "9mm"
+	max_ammo = 30
+
+/obj/item/ammo_box/magazine/spitter_9mm/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state]-[!!ammo_count()]"
+
+
+/obj/item/gun/ballistic/automatic/smg/pounder
+	name = "Pounder"
+	desc = "A strange frontiersmen weapon. With a high ammo count and a low caliber, this gun makes up for its lack of power with it's extremely high rate of fire, hence the name. Chambered in .22 LR."
+	icon = 'icons/obj/guns/manufacturer/frontier_import/48x32.dmi'
+	lefthand_file = 'icons/obj/guns/manufacturer/frontier_import/lefthand.dmi'
+	righthand_file = 'icons/obj/guns/manufacturer/frontier_import/righthand.dmi'
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/frontier_import/onmob.dmi'
+
+	icon_state = "pounder"
+	item_state = "pounder"
+	mag_type = /obj/item/ammo_box/magazine/c22lr_pounder_pan
+	burst_size = 1
+	fire_delay = 0.05 SECONDS
+	spread = 25
+	spread_unwielded = 50
+
+	fire_sound = 'sound/weapons/gun/smg/pounder.ogg'
+	rack_sound = 'sound/weapons/gun/smg/pounder_cocked.ogg'
+	rack_sound_vary = FALSE
+
+	load_sound_vary = FALSE
+	eject_sound_vary = FALSE
+	load_sound = 'sound/weapons/gun/smg/pounder_reload.ogg'
+	load_empty_sound = 'sound/weapons/gun/smg/pounder_reload.ogg'
+	eject_sound = 'sound/weapons/gun/smg/pounder_unload.ogg'
+	eject_empty_sound = 'sound/weapons/gun/smg/pounder_unload.ogg'
+
+	gun_firemodes = list(FIREMODE_FULLAUTO)
+	default_firemode = FIREMODE_FULLAUTO
+
+	manufacturer = MANUFACTURER_IMPORT
+	wield_slowdown = 0.5
+
+/obj/item/ammo_box/magazine/c22lr_pounder_pan
+	name = "pan magazine (.22 LR)"
+	desc = "A 50-round pan magazine for the Pounder machine gun. It's rather tiny, all things considered; it looks like it wouldn't pierce a single piece of armor."
+	icon_state = "firestorm_pan"
+	base_icon_state = "firestorm_pan"
+	ammo_type = /obj/item/ammo_casing/c22lr
+	caliber = "22lr"
+	max_ammo = 50
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/ammo_box/magazine/c22lr_pounder_pan/update_icon_state()
+	. = ..()
+	icon_state = "firestorm_pan"
+
+/obj/item/gun/ballistic/automatic/hmg/shredder
+	name = "\improper Shredder"
+	desc = "A peculiar Frontiersman heavy machine gun. The tripod has been removed, a chainsaw grip for hip fire at the front has been added, and it has been rechambered to fire shotgun shells. Chambered in 12g."
```

Comment:
"A vastly atypical heavy machine gun, extensively modified by the Frontiersmen. Additional grips have been added to enable firing from the hip, and it has been modified to fire belts of shotgun shells. Chambered in 12g."